lib/active_shipping/carriers/fedex.rb

```diff
@@ -136,6 +136,9 @@ class FedEx < Carrier
 
     DEFAULT_LABEL_STOCK_TYPE = 'PAPER_7X4.75'
 
+    # Available return formats for image data when creating labels
+    LABEL_FORMATS = ['DPL', 'EPL2', 'PDF', 'ZPLII', 'PNG'] 
```

Comment:
More as a quick reference for API consumers so they don't have to wade through the specs.  I considered a more robust API here (input validation, eg), but I'm following the general coding practices of the gem which are to be very loose on the inputs accepted.